app/src/main/res/layout/admin_auth_activity.xml

```diff
@@ -40,91 +40,78 @@
-          app:layout_constraintTop_toTopOf="parent" />
+    <TextView
+      android:id="@+id/admin_auth_heading_textview"
+      style="@style/Heading2"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="28dp"
+      android:layout_marginTop="12dp"
+      android:layout_marginEnd="28dp"
+      android:maxLines="2"
+      android:text="@string/admin_auth_heading"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@id/admin_auth_app_bar_layout" />
 
-        <TextView
-          android:id="@+id/admin_auth_sub_text"
-          android:layout_width="0dp"
-          android:layout_height="wrap_content"
-          android:layout_marginStart="28dp"
-          android:layout_marginTop="12dp"
-          android:layout_marginEnd="28dp"
-          android:fontFamily="sans-serif"
-          android:text="@string/admin_auth_sub"
-          android:textColor="@color/oppiaAdminAuthSecondaryText"
-          android:textSize="16sp"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toBottomOf="@id/admin_auth_heading_textview" />
+    <TextView
+      android:id="@+id/admin_auth_sub_text"
+      style="@style/Subtitle1"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="28dp"
+      android:layout_marginTop="12dp"
+      android:layout_marginEnd="28dp"
+      android:text="@string/admin_auth_sub"
+      android:textColor="@color/oppiaAdminAuthSecondaryText"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@id/admin_auth_heading_textview" />
 
-        <org.oppia.android.app.profile.ProfileInputView
-          android:id="@+id/admin_auth_input_pin"
-          android:layout_width="match_parent"
-          android:layout_height="wrap_content"
-          android:layout_marginStart="28dp"
-          android:layout_marginTop="48dp"
-          android:layout_marginEnd="28dp"
-          android:addTextChangedListener="@{viewModel.inputPinTextWatcher}"
-          android:imeOptions="actionDone"
-          app:inputLength="5"
-          app:isPasswordInput="true"
-          app:label="@string/admin_auth_pin"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toBottomOf="@id/admin_auth_sub_text"
-          profile:error="@{viewModel.errorMessage}"
-          profile:labelMargin="@{@dimen/space_8dp}" />
+    <org.oppia.android.app.profile.ProfileInputView
+      android:id="@+id/admin_auth_input_pin"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="28dp"
+      android:layout_marginTop="24dp"
```

Comment:
This margin has been updated, keep it same as before `48dp`.

Reply:
@bladebunny This is pending

Reply:
Done